mivideo/mivideo_extract.py

```diff
@@ -2,29 +2,37 @@
 '''
 Module for setting up and running the MiVideo data extract.
 '''
-
 import logging
 import os
 import time
-from datetime import datetime, timezone
-from typing import Dict, Sequence, Union
+from datetime import datetime
+from typing import Dict, Iterable, Sequence, Union
 
 import pandas as pd
+from KalturaClient import KalturaClient, KalturaConfiguration
+from KalturaClient.Plugins.Core import (
```

Comment:
Interesting, I haven't seen this import syntax before.

Reply:
I don't like backslash continuation for various reasons.  I've learned that Python is happy with parenthesized expressions that contain line breaks.  I hadn't used this syntax with `import` before, but I thought I'd give it a try.  It worked.